### PR TITLE
Remove checklist

### LINKED ISSUE #837: Problem: formatting issues in Working with Archivematica docs
**Describe the bug**

There are some formatting issues in [Working with Archivematica](Working with Archivematica
https://enduro.readthedocs.io/dev-manual/archivematica/) introduced by https://github.com/artefactual-sdps/enduro/pull/828 and https://github.com/artefactual-sdps/enduro/pull/830.

**To Reproduce**

1. Visit https://enduro.readthedocs.io/dev-manual/archivematica/.
2. Look at the formatting issues.
3. Experience visual discomfort.
4. Feel the urge to fix ASAP.

### PR DESCRIPTION
**Pull request by Diogenesoftoronto**
- Removes the checklist that had incorrect formatting with mkdocs.
- Makes the documentation formatting more consistent with the rest of the documentation.
- Replaces the checklist with headers.
- Uses sentence case for headers greater than two.

Closes https://github.com/artefactual-sdps/enduro/issues/837.

### CODE REVIEW

**Review by sevein — CHANGES_REQUESTED** [review]
It doesn't look very good.

Have you tried it? The preview site is available at: https://enduro--839.org.readthedocs.build/dev-manual/archivematica/

**Comment by Diogenesoftoronto** [comment]
> It doesn't look very good.
> 
> Have you tried it? The preview site is available at: [enduro--839.org.readthedocs.build/dev-manual/archivematica](https://enduro--839.org.readthedocs.build/dev-manual/archivematica/)

I thought it looked okay, would you prefer subheaders instead?

File: docs/src/dev-manual/archivematica.md [hunk 1]
```diff
@@ -20,34 +20,43 @@ inside the cluster, and they are not tracked in the repository.
 
 ### Quick Checklist for Configuration Files
```
Comment (sevein) [inline docs/src/dev-manual/archivematica.md]: Sentence case capitalization:

```suggestion
### Quick checklist for configuration files
```

File: docs/src/dev-manual/archivematica.md [hunk 2]
```diff
@@ -20,34 +20,43 @@ inside the cluster, and they are not tracked in the repository.
...
-  - **Contents to Check:**
-    - ENDURO_PRES_SYSTEM = "am"
-- [ ] enduro.toml File:
-  - Location: `root/`
-  - **Contents to Check:**
-    - Preservation Taskqueue variable must be set to "am"
+#### .am.secret File
+
+- Location: `hack/kube/overlays/dev-am/.am.secret`
+- **Contents to Check:**
```
Comment (sevein) [inline docs/src/dev-manual/archivematica.md]: Sentence case capitalization:

```suggestion
- **Contents to check:**
```

File: docs/src/dev-manual/archivematica.md [hunk 3]
```diff
@@ -20,34 +20,43 @@ inside the cluster, and they are not tracked in the repository.
...
+  - AM API address (e.g.,`http://host.k3d.internal:62080`)
+  - User credentials(`user=test`, `api_key=test`)
```
Comment (sevein) [inline docs/src/dev-manual/archivematica.md]: Missing space:

```suggestion
  - User credentials (`user=test`, `api_key=test`)
```

File: docs/src/dev-manual/archivematica.md [hunk 4]
```diff
@@ -20,34 +20,43 @@ inside the cluster, and they are not tracked in the repository.
...
+  OPENSSH PRIVATE KEY-----` and ends with `-----END
+  OPENSSH PRIVATE KEY-----`)
+
+#### .known_hosts.secret File
+
+- Location: `hack/kube/overlays/dev-am/.known_hosts.secret`
+- **Contents to Check:**
+  - Known hosts entries (Look for entries starting with
+  `|1|` and containing `ssh-rsa`, `ecdsa-sha2-nistp256`,
+  `ssh-ed25519` etc.)
```
Comment (sevein) [inline docs/src/dev-manual/archivematica.md]: All lists in this PR seem to share the same problem with nesting lists, but I'm using this section as the example. It renders as follows:

image

Given how you indented the code I suspect that wasn't your intention. I believe that the list items under "Contents to check" need to use 4-space indentation in mkdocs. After the change:

```suggestion
#### .known_hosts.secret File

- Location: `hack/kube/overlays/dev-am/.known_hosts.secret`
- **Contents to Check:**
    - Known hosts entries (Look for entries starting with
      `|1|` and containing `ssh-rsa`, `ecdsa-sha2-nistp256`,
      `ssh-ed25519` etc.)
```

It renders as follows:

image

File: docs/src/dev-manual/archivematica.md [hunk 5]
```diff
@@ -20,34 +20,43 @@ inside the cluster, and they are not tracked in the repository.
...
+
+#### .tilt.env File
+
+- Location: `root/`
+- **Contents to Check:**
+  - ENDURO_PRES_SYSTEM = "am"
```
Comment (sevein) [inline docs/src/dev-manual/archivematica.md]: This should be enclosed in backticks just like the other examples.

File: docs/src/dev-manual/archivematica.md [hunk 6]
```diff
@@ -20,34 +20,43 @@ inside the cluster, and they are not tracked in the repository.
...
+
+#### enduro.toml File
```
Comment (sevein) [inline docs/src/dev-manual/archivematica.md]: Sentence case capitalization:

```suggestion
#### enduro.toml file
```

But for consistency with the section below I'd suggest:

```suggestion
#### `enduro.toml`
```

**Review by sevein — CHANGES_REQUESTED** [review]
I've added a few additional comments. I apologize if this appears overly meticulous. It's really helpful if you build the documents locally if you haven't yet. There are a few minor issues to address, such as a missing space, inconsistent styles of title capitalization, and a potential issue with how we're indenting lists.

I've also noticed that the pull request doesn't link back to the issue you're addressing.

Thank you!

**Comment by sevein** [comment]
> I thought it looked okay, would you prefer subheaders instead?

Maybe we weren't looking at the same place? GitHub was nesting lists alright but MkDocs wasn't because it expects 4-space indentation. I took a screenshot of the state of the document generated by MkDocs before you added the headers that made me bring up the issue:

image

File: docs/src/dev-manual/archivematica.md (hunk 1 above)
Comment (Diogenesoftoronto) [reply docs/src/dev-manual/archivematica.md]: For a header? Header should have title case no? They are titles.

File: docs/src/dev-manual/archivematica.md (hunk 4 above)
Comment (Diogenesoftoronto) [reply docs/src/dev-manual/archivematica.md]: I will check markdownlint to see if this doesn't throw errors. That's my initial worry with this.

**Comment by Diogenesoftoronto** [comment]
closes #837

**Comment by Diogenesoftoronto** [comment]
This breaks markdownlint-cli rule  md007, I suggest we ignore or modify this rule for all documentation.

**Review by sevein — APPROVED** [review]

**Pull request merged**